rclpy/CMakeLists.txt

```diff
@@ -175,7 +175,7 @@ if(BUILD_TESTING)
       test/test_messages.py
       test/test_node.py
       test/test_parameter.py
-      test/test_parameter_client.py
+      test/test_parameter_service.py
```

Comment:
you don't need to remove that

```suggestion
      test/test_parameter_client.py
      test/test_parameter_service.py
```

Reply:
@ivanpauno Friendly ping. Please look at above my comments.

Reply:
it should be added, right? i think we can keep this file as well.

Reply:
Okay. Keep it

Reply:
yes, we can keep it